language-server/modules/langserver-core/src/test/resources/completion/function/chainCompletion4.json

```diff
@@ -4,358 +4,336 @@
     "character": 7
   },
   "source": "function/source/chainCompletion4.bal",
-  "items": [
-    {
-      "label": "shift()((any|error))",
-      "kind": "Function",
-      "detail": "Function",
-      "documentation": {
-        "right": {
-          "kind": "markdown",
-          "value": "**Package:** _ballerina/lang.array_  \n  \nRemoves and returns first member of an array.\nThe array must not be empty.\n  \n  \n  \n**Returns** `(any|error)`   \n- the value that was the first member of the array  \n  \n"
-        }
-      },
-      "sortText": "120",
-      "insertText": "shift()",
-      "insertTextFormat": "Snippet"
+  "items": [{
+    "label": "shift()((any|error))",
+    "kind": "Function",
+    "detail": "Function",
+    "documentation": {
+      "right": {
+        "kind": "markdown",
+        "value": "**Package:** _ballerina/lang.array_  \n  \nRemoves and returns first member of an array.\nThe array must not be empty.\n  \n  \n  \n**Returns** `(any|error)`   \n- the value that was the first member of the array  \n  \n"
+      }
     },
-    {
-      "label": "toStream()(stream<(anydata|error)><(anydata|error)>)",
-      "kind": "Function",
-      "detail": "Function",
-      "documentation": {
-        "right": {
-          "kind": "markdown",
-          "value": "**Package:** _ballerina/lang.array_  \n  \nReturns a stream from the given array.\n  \n"
-        }
-      },
-      "sortText": "120",
-      "insertText": "toStream()",
-      "insertTextFormat": "Snippet"
+    "sortText": "120",
+    "insertText": "shift()",
+    "insertTextFormat": "Snippet"
+  }, {
```

Comment:
Shall we format the JSON using https://jsonformatter.curiousconcept.com/ and paste it here. Ideally, we need to add only the specific change. Most of these changes are just indentation.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/21275/commits/44296d135055b7826de9430384aaec4b1cbd9270